src/AhRulesBot/AhRulesBot/BotRequestsProcessing/Handlers/CustomRulesMessageHandler.cs

```diff
@@ -39,7 +39,7 @@ private List<string> ProcessRulesRequest(string message)
             bool messageContains(CustomRuleItem item) => item.Title.Contains(message, StringComparison.InvariantCultureIgnoreCase);
             bool messageExact(CustomRuleItem item) => item.Title.Equals(message, StringComparison.InvariantCultureIgnoreCase);
 
-            while (_channel.TryRead(out List<CustomRuleItem> rulesUpdate) && rulesUpdate != null)
+            while (_channel.TryRead(out List<CustomRuleItem>? rulesUpdate) && rulesUpdate != null)
```

Comment:
but if you only access that variable if it's not null, should the type of it really be nullable?